docs/docs/Get-Started/get-started-quickstart.md

```diff
@@ -4,188 +4,502 @@ slug: /get-started-quickstart
+                },
+                "icon": "bot",
+                "allow_markdown": false,
+                "positive_feedback": null,
+                "state": "complete",
+                "targets": []
+              },
+              "category": "message",
+              "content_blocks": [
+                {
+                  "title": "Agent Steps",
+                  "contents": [
+                    {
+                      "type": "text",
+                      "duration": 2,
+                      "header": {
+                        "title": "Input",
+                        "icon": "MessageSquare"
+                      },
+                      "text": "**Input**: hello world!"
+                    },
+                    {
+                      "type": "text",
+                      "duration": 226,
+                      "header": {
+                        "title": "Output",
+                        "icon": "MessageSquare"
+                      },
+                      "text": "Hello world! 🌍 How can I assist you today?"
+                    }
+                  ],
+                  "allow_markdown": true,
+                  "media_url": null
+                }
+              ],
+              "duration": null
+            }
+          },
+          "artifacts": {
+            "message": "Hello world! 🌍 How can I assist you today?",
+            "sender": "Machine",
+            "sender_name": "AI",
+            "files": [],
+            "type": "object"
+          },
+          "outputs": {
+            "message": {
+              "message": "Hello world! 🌍 How can I assist you today?",
+              "type": "text"
+            }
+          },
+          "logs": {
+            "message": []
+          },
+          "messages": [
+            {
+              "message": "Hello world! 🌍 How can I assist you today?",
+              "sender": "Machine",
+              "sender_name": "AI",
+              "session_id": "29deb764-af3f-4d7d-94a0-47491ed241d6",
+              "stream_url": null,
+              "component_id": "ChatOutput-aF5lw",
+              "files": [],
+              "type": "text"
+            }
+          ],
+          "timedelta": null,
+          "duration": null,
+          "component_display_name": "Chat Output",
+          "component_id": "ChatOutput-aF5lw",
+          "used_frozen_result": false
+        }
+      ]
+    }
+  ]
+}
+```
 
-### Run the chatbot with retrieved context
+</details>
+
+This response confirms the call succeeded, but let's do something more with the returned answer from the agent.
+
+The following example builds on the API pane's example code to create a question-and-answer chat in your terminal that stores the Agent's previous answer.
+
+3. Copy the code into your terminal, and run it.
+4. To view the Agent's previous answer, type `compare`. To close the terminal chat, type `exit`.
+
+<Tabs groupId="Languages">
+  <TabItem value="Python" label="Python" default>
+
+```python
```

Comment:
I would modify these snippets to omit the Simple Agent snippet or replace the unique pieces with placeholders. If they try to run your scripts as-is, they will fail because of different base urls and flow IDs.

I think it would be helpful to use comments or something to highlight where in the script you incorporate the pieces of the Langflow snippet. I was scrolling back and forth so I could compare them.